resources/aws_ec2.go

```diff
@@ -111,6 +113,36 @@ const (
 	awsEc2EventInstanceExists
 )
 
+// bootstrapScript is a bash script template for installing mgmt and connecting
+// back to the seed specified in the definition.
+const bootstrapScript string = `#!/bin/bash
+if [ -n "$(command -v apt)" ]
+then
+  apt update
+  apt install -y libaugeas0 libvirt0
+fi
+
+if [ -n "$(command -v yum)" ]
+then
+  yum install -y augeas-libs libvirt-libs
+fi
+
+mkdir -p "/opt/mgmtconfig"
+curl "{{- .Binary -}}" -o "/opt/mgmtconfig/mgmt"
+chmod +x "/opt/mgmtconfig/mgmt"
+
+curl "{{- .RemoteGraph -}}" -o graph.yaml
+
+internal=$(curl http://169.254.169.254/latest/meta-data/local-ipv4)
+external=$(curl http://169.254.169.254/latest/meta-data/public-ipv4)
+
+/opt/mgmtconfig/mgmt run --yaml graph.yaml --no-pgp --tmp-prefix \
+--seeds {{.Seed}} \
+--client-urls "http://${internal}:2379" \
```

Comment:
Should we let the user define the client and server ports?

Reply:
Good question. Let's say no for now. You can pull the values we want to use from etcd package (they are constants in our etcd package) https://github.com/purpleidea/mgmt/blob/master/etcd/etcd.go#L90
You can either split on these, or patch it to store them separately in etcd.go -- I think parsing is preferable.

Reply:
Unfortunately it's not possible since etcd imports resources so importing etcd isn't allowed. I added them as consts.